cfme/tests/automate/test_instance.py

```diff
@@ -219,3 +219,59 @@ def test_automate_relationship_trailing_spaces(request, klass, namespace, domain
             assert result.output == ""
         else:
             assert search in result.output
+
+
+@pytest.fixture(scope="module")
+def copy_instance(domain):
+    """
+    This fixture copies the instance '/ManageIQ/System/Request/ansible_tower_job' to new domain.
+    """
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="System"
+    ).classes.instantiate(name="Request").instances.instantiate(name="ansible_tower_job").copy_to(
+        domain.name
+    )
+    instance = (
+        domain.namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+        .instances.instantiate(name="ansible_tower_job")
+    )
+    yield instance
+
+
+@pytest.mark.tier(1)
+def test_check_system_request_calls_depr_configurationmanagement(appliance, copy_instance):
```

Comment:
Optional: Consider shortening the name of this test case to `test_check_system_request_calls_depr_conf_mgmt`